src/core/request/deltafetch.js

```diff
@@ -190,4 +122,26 @@ export class DeltaFetchRequest extends KinveyRequest {
 
     return this.url.substring(ind + appkeyStr.length, indOfQueryString);
   }
+
+  static execute(options, client, dataOnly = true) {
```

Comment:
Isn't this inherited?

Reply:
It is but the inherited function will do `new KinveyRequest()` and not `new DeltaFetchRequest()`.